pkg/meta/base.go

```diff
@@ -591,6 +591,9 @@ func (m *baseMeta) Mknod(ctx Context, parent Ino, name string, _type uint8, mode
 	if m.conf.ReadOnly {
 		return syscall.EROFS
 	}
+	if name == "" {
+		return syscall.ENOENT
```

Comment:
EINVAL

Reply:
It's what returned by glibc creat/rename functions. Also os.Create/os.Rename return ENOENT as well.